cpp/ql/src/Critical/InconsistentNullnessTesting.ql

```diff
@@ -5,6 +5,7 @@
  * @id cpp/inconsistent-nullness-testing
  * @problem.severity warning
  * @tags reliability
+ *       security
```

Comment:
I know that dereferencing null is UB, but how often does it actually cause a security problem?